app/styles/layout/_layout.scss

```diff
@@ -72,4 +72,8 @@ MAIN {
   .text-muted {
     color: $mid-grey;
   }
+
+  .bg-disabled {
```

Comment:
`.bg-disabled` class exists in `styles/base/_coloer.scss` don't know why you need this?

Reply:
To make sure the background color is grey.  If not, the background color is white.
![image](https://user-images.githubusercontent.com/18737885/51222315-27710900-1978-11e9-91c7-732ed0cc9ebe.png)